trade-finance-manager-ui/server/controllers/case/amendments/amendmentBankReviewDate.post.api-test.js

```diff
@@ -22,6 +22,8 @@ const user = {
 
 const session = { user, userToken: 'mockToken' };
 
+const gefFacilityType = 'Cash facility';
```

Comment:
does 'Cash facility' exist in constants? If it does, can the constant be used here

Reply:
No, it doesn't unfortunately

Reply:
could a constant be created and used across all these files which are using 'Cash facility' please?

Reply:
Constant file made, and implemented everywhere